server/accounts.go

```diff
@@ -4281,25 +4412,33 @@ func newTransform(src, dest string) (*transform, error) {
 
 		nphs := 0
 		for _, token := range dtokens {
-			tp, nb, err := placeHolderIndex(token)
+			tranformType, transformArgWildcardIndexes, transfomArgInt, transformArgString, err := indexPlaceHolders(token)
 			if err != nil {
 				return nil, err
 			}
-			if tp[0] >= 0 {
+
+			if tranformType == NoTransform {
+				{
```

Comment:
needed?

Reply:
The short answer is yes

Reply:
how is the `{` on line `4421` and the closing `}` on line `4426` relevant?